### ISSUE TITLE
Possible to support libsixel

### ISSUE THREAD

**Issue by HalosGhost** [martanne/dvtm#8]
As I grow more and more disenchanted with X, I am more and more enticed by the notion of moving to a workflow that consists of [``yaft``](https://github.com/uobikiemukot/yaft), ``abduco`` and ``dvtm``. However, one thing I love about ``yaft`` is its support for [``libsixel``](https://github.com/saitoha/sixel). This allows me to keep whatever small graphical things around I might still want.

Is there any chance that dvtm (and abduco, if applicable) could get support for libsixel?

**Comment by martanne** [comment]
I agree this could potentially be interesting. I'm not sure how it would integrate with curses though. Patches welcome. 

In general dvtm's terminal emulation code was not really written to the spec and could probably be improved. In the long term I wouldn't really mind if I did not have to maintain it. Thus it might be worth considering to reuse the terminal emulation code of [st](http://st.suckless.org/) or build up on [libvterm](http://www.leonerd.org.uk/code/libvterm/).

I will reconsider sixel support when deciding what to do, but for now I unfortunately won't have time.

**Comment by HalosGhost** [comment]
Oh my; if it is at all possible for both ``st`` and ``dvtm`` to get libsixel support, that would be a dream come true.

I do not have much personal experience with libsixel, but I will look into what level of work would be required and perhaps work out a patch. I'm excited to see someone else interested in the notion!
